Practical7/src/store/index.js

```diff
@@ -0,0 +1,173 @@
+import { createStore } from "vuex";
```

Comment:
make different modules here.

Reply:
This different made on next practical 8 as you have told me.